capsules/src/qdec.rs

```diff
@@ -0,0 +1,113 @@
+//! Provides userspace access to the Qdec on a board.
+//!
+//! Usage
+//! -----
+//!
+//! ````
+//! let qdec = static_init!(
+//!     capsules::qdec::Qdec<'static>,
+//!     capsules::qdec::QdecInterface::new(&nrf52::qdec::QDEC,
+//!                                         kernel::Grant::create())
+//! );
+//! kernel::hil::QdecDriver.set_client(qdec);
+//! ````
+//!
+//! #Number of Qdecs
+//! The number of qdecs available depends on the
+//! board/microcontroller
```

Comment:
What does this refer to? If this capsule doesn't support configuring multiple QDECs I don't think you need to include this